.github/workflows/markdown-link-check-config.json

```diff
@@ -1,13 +1,30 @@
 {
   "ignorePatterns": [
     {
-      "pattern": "^https://.*\.visualstudio\.com"
+      "pattern": "^https:\/\/.*\\.visualstudio\\.com"
+    },
+    {
+      "pattern": "^https:\/\/dev\\.azure\\.com"
+    },
+    {
+      "pattern": "^https:\/\/aka\\.ms"
+    },
+    {
+      "pattern": ".*❓.*"
     }
   ],
   "replacementPatterns": [
     {
       "pattern": "^/",
       "replacement": "{{BASEURL}}/"
     }
+  ],
+  "httpHeaders": [
+    {
+      "urls": ["https://docs.github.com/"],
+      "headers": {
+        "Accept-Encoding": "zstd, br, gzip, deflate"
+      }
+    }
```

Comment:
Workaround for `docs.github.com` links. See: https://github.com/tcort/markdown-link-check/issues/201#issuecomment-1110242146